src/backend/ref/dh-newhope.c

```diff
@@ -176,8 +176,10 @@ NoiseDHState *noise_newhope_new(void)
     return &(state->parent);
 }
 
+#if !USE_LIBSODIUM
 /* Implementation of random number generation needed by New Hope */
 void randombytes(unsigned char *x,unsigned long long xlen)
```

Comment:
I assume this is because libsodium already has a function called randombytes()?

We can probably remove this whole function.  I modified the API to newhope_keygen() and newhope_sharedb() to take the random data in as an argument.  I left the randombytes() callback in newhope.c for if the caller passed in a NULL buffer.  That could be removed entirely.